railgun/internal/diagnostics/server.go

```diff
@@ -46,8 +63,22 @@ func (s *Server) Listen(ctx context.Context) error {
 	}
 }
 
-// installHandlers adds the Profiling webservice to the given mux.
-func installHandlers(mux *http.ServeMux) {
+// receiveConfig watches the config update channel
+func (s *Server) receiveConfig(ctx context.Context) {
+	for {
+		select {
+		case dump := <-s.ConfigDumps.SuccessfulConfigs:
+			successfulConfigDump = dump
+		case dump := <-s.ConfigDumps.FailedConfigs:
+			failedConfigDump = dump
```

Comment:
case failedConfigDump := <-s.ConfigDumps.FailedConfigs:

Reply:
Obsoleted by changes in d669d6b8